tests/template/logparser_data/cloudmc1.txt

```diff
@@ -0,0 +1,134 @@
+                ###################     %%%%%%%%%%%%%%%%%%%%%%%
+                ####################     %%%%%%%%%%%%%%%%%%%%%%
+                #####################     %%%%%%%%%%%%%%%%%%%%%
+                ######################     %%%%%%%%%%%%%%%%%%%%
+                #######################     %%%%%%%%%%%%%%%%%%
+                 #######################     %%%%%%%%%%%%%%%%%
+                 ######################     %%%%%%%%%%%%%%%%%
+                  ####################     %%%%%%%%%%%%%%%%%
+                    #################     %%%%%%%%%%%%%%%%%
+                     ###############     %%%%%%%%%%%%%%%%
+                       ############     %%%%%%%%%%%%%%%
+                          ########     %%%%%%%%%%%%%%
+                                      %%%%%%%%%%%
+
+                 | The OpenMC Monte Carlo Code
+       Copyright | 2011-2023 MIT, UChicago Argonne LLC, and contributors
+         License | https://docs.openmc.org/en/latest/license.html
+         Version | 0.13.4-dev
+        Git SHA1 | eea52238dac0162a8b7924b81800af4a3c78798e
+       Date/Time | 2024-01-26 20:16:33
+   MPI Processes | 1
+  OpenMP Threads | 6
+
+ Reading settings XML file...
+ Reading cross sections XML file...
+ Reading materials XML file...
+ Reading geometry XML file...
+Loading file geometryInput-dagmcFile.aurora.h5m
+Initializing the GeomQueryTool...
+Using faceting tolerance: 0.0001
+Building acceleration data structures...
+ Reading Cu63 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_Cu63.h5
+ Reading Cu65 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_Cu65.h5
+ Reading N14 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_N14.h5
+ Reading N15 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_N15.h5
+ Reading O16 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_O16.h5
+ Reading Ar36 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_Ar36.h5
+ WARNING: Negative value(s) found on probability table for nuclide Ar36 at 294K
+ Reading Ar38 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_Ar38.h5
+ Reading Ar40 from
+ /home/vagrant/src/radiasoft/sirepo/run/user/deXWKOcN/tmp/cloudmc-cache/ENDFB-7.1
+ -NNDC_Ar40.h5
+ Minimum neutron data temperature: 294 K
+ Maximum neutron data temperature: 294 K
+ Reading tallies XML file...
+ Preparing distributed cell instances...
+ Reading plot XML file...
+ Maximum neutron transport energy: 20000000 eV for N15
+ Initializing source particles...
+
+ ====================>     K EIGENVALUE SIMULATION     <====================
+
+  Bat./Gen.      k            Average k
+  =========   ========   ====================
+ ERROR: No fission sites banked on MPI rank 0
```

Comment:
@moellep  is there value in keeping the pieces of the log above this line?

Reply:
Not in this case - just the "ERROR: No fission sites banked on MPI rank 0" line.